ibllib/pipes/training_preprocessing.py

```diff
@@ -31,32 +29,23 @@ def _run(self):
         """
         Extracts an iblrig training session
         """
-        trials, wheel, output_files = extract_training(self.session_path, save=True)
+        trials, wheel, output_files = bpod_trials.extract_all(self.session_path, save=True)
         if trials is None:
             return None
 
         # Run the task QC
         # Compile task data for QC
-        def _qc_extract(qc):
-            qc.extractor = TaskQCExtractor(self.session_path, lazy=True, one=qc.one)
-            qc.extractor.data = trials
-            qc.extractor.extract_data(partial=True)  # Extract the rest of the data
-            return qc
-
-        if get_session_extractor_type(self.session_path) == 'habituation':
+        type = get_session_extractor_type(self.session_path)
+        if type == 'habituation':
             qc = HabituationQC(self.session_path, one=self.one)
-            qc = _qc_extract(qc)
+            qc.extractor = TaskQCExtractor(self.session_path, one=self.one)
```

Comment:
The lazy option would miss adding the `quiescence` in the dictionary and omit the `self.settings` property